tools/build_defs/cc/tests/cc_import_test.bzl

```diff
@@ -104,10 +104,36 @@ def _cc_import_objects_archive_action_test_impl(ctx):
 
     asserts.true(env, found, "Archive creation action should be present if object files were given")
 
+    linker_inputs = target_under_test[CcInfo].linking_context.linker_inputs.to_list()
```

Comment:
Can you add a space between each block of 2 lines with assignment + assert? I think that will make it easier to read.